src/resources/js/components/Authentication.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="Container">
+  
+    <h2>Authentification</h2>
+
+    <div v-if="isMessage">
+      <h3>{{ message }}</h3>
+    </div>
+
+    <form @submit.prevent="login()">
+      <label for="EnterEMail">
+        <input id="EnterEMail" type="text" v-model="input.email" placeholder="Adresse e-mail" />
+      </label>
+
+      <label for="EnterPassword">
+        <input id="EnterPassword" type="password" v-model="input.password" placeholder="Mot de passe" />
+      </label>
+
+      <input type="submit" value="Se connecter"/>
+      
+
+    </form>
+    
+
+  </div>
+</template>
+
+<script>
+  import axios from 'axios'
+  import router from '../routes'
+
+  export default {
+  name : 'Authentication',
+  props:['parent'], //for future use (can't log in if already logged in, will reroute to home)
+  data () {
+    return{
+      input: {
+        email: '',
+        password: ''
+      },
+      isMessage: false,
+      message: ''
+    }
+
+
+  },
+  methods: {
+    login() {
+      if(this.input.email != "" && this.input.password != "") { //if not blank
+        //sends credentials to backend for verification
+
+        if(!this.validateEmail(this.input.email)){
+          this.message='Addresse e-mail non valide';
+          this.isMessage=true;
+          return;
+        }
+
+        axios.post(
+          '/api/login',{ // might change, depend of back end route
+            email: this.input.email,
+            password: this.input.password
+          }
+              
+        )
+        .then(function (data){
```

Comment:
guidelines: use arrow syntax for anonymous functions, [source](https://github.com/airbnb/javascript#arrows--use-them)

Reply:
it has been changed